src/pages/documentation/mobile-core/signal/tabs/api-reference/register-extension/ios-aep.md

```diff
@@ -0,0 +1 @@
+This API no longer exists in `Signal`. Instead, the extension should be registered by calling the `registerExtensions` API in the MobileCore. Please see the updated SDK initialization steps at the [migrate to Swift tutorial.](../../resources/migrate-to-swift.md#update-sdk-initialization) FIX LINK
```

Comment:
Does this link need to be fixed?

Reply:
Me putting FIX LINK (like here) is generally for ones where I don't know exactly where it'll live on the new-side just yet (which is why I have a reminder for me to do it later).